eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -54,6 +60,20 @@ def create_enrollment(*args, **kwargs):
     return enrollment, errors
 
 
+def create_program_enrollment(program_uuid, *arg, **kwargs):
+    """
+    backend function to create enrollment
+    """
+    try:
+        site = models.ForeignKey(Site)
+        data = site.siteconfiguration.get_program(program_uuid)
+        for x in data['courses']:
+            print(x)
+            pprint(vars(x))
```

Comment:
I suppose here we are going to do the enrollments one by one. Maybe this should also call the `enroll_on_course` function at some point

Reply:
Yeah that was the idea